R/datasets.R

```diff
@@ -0,0 +1,640 @@
+#' @return A dataframe.
+#' @export
+#' @md
+load_dataset_into_data_frame <- function(dataset)	{
+  dataset$to_pandas_data_frame()
+}
+
+#' Convert the current dataset into a FileDataset containing CSV files.
+#'
+#' @description
+#' Convert the current dataset into a FileDataset containing CSV files.
+#'
+#' @param dataset The Tabular Dataset object.
+#' @param separator The separator to use to separate values in the resulting file.
+#' @return A new FileDataset object with a set of CSV files containing the data
+#' in this dataset.
+#' @export
+#' @md
+
+convert_to_dataset_with_csv_files <- function(dataset, separator = ",") {
+  dataset$to_csv_files(separator)
+}
+
+#' Convert the current dataset into a FileDataset containing Parquet files.
+#'
+#' @description
+#' Convert the current dataset into a FileDataset containing Parquet files.
+#' The resulting dataset will contain one or more Parquet files, each corresponding
+#' to a partition of data from the current dataset. These files are not materialized
+#' until they are downloaded or read from.
+#'
+#' @param dataset The Tabular Dataset object.
+#' @return A new FileDataset object with a set of Parquet files containing the
+#' data in this dataset.
+#' @export
+#' @md
+convert_to_dataset_with_parquet_files <- function(dataset) {
+  dataset$to_parquet_files()
+}
+
+#' Configure conversion to bool.
+#'
+#' @description
+#' Configure conversion to bool.
+#'
+#' @return Converted DataType object.
+#' @export
+#' @md
+data_type_bool <- function() {
+  azureml$data$dataset_factory$DataType$to_bool()
+}
+
+#' Configure conversion to datetime.
+#'
+#' @description
+#' Configure conversion to datetime.
+#'
+#' @param formats Formats to try for datetime conversion. For example `%d-%m-%Y` for data in "day-month-year",
+#' and `%Y-%m-%dT%H:%M:%S.%f` for "combined date an time representation" according to ISO 8601.
+#' * %Y: Year with 4 digits
+#' * %y: Year with 2 digits
+#' * %m: Month in digits
+#' * %b: Month represented by its abbreviated name in 3 letters, like Aug
+#' * %B: Month represented by its full name, like August
+#' * %d: Day in digits
+#' * %H: Hour as represented in 24-hour clock time
+#' * %I: Hour as represented in 12-hour clock time
+#' * %M: Minute in 2 digits
+#' * %S: Second in 2 digits
+#' * %f: Microsecond
+#' * %p: AM/PM designator
+#' * %z: Timezone, for example: -0700
+#'
+#' Format specifiers will be inferred if not specified.
+#' Inference requires that the data source is accessible from current compute.
+#' @return Converted DataType object.
+#' @export
+#' @md
+data_type_datetime <- function(formats = NULL) {
+  azureml$data$dataset_factory$DataType$to_datetime(formats)
+}
+
+#' Configure conversion to 53-bit double.
+#'
+#' @description
+#' Configure conversion to 53-bit double.
+#'
+#' @return Converted DataType object.
+#' @export
+#' @md
+data_type_double <- function()	{
+  as.double(azureml$data$dataset_factory$DataType$to_float())
```

Comment:
actually, all these methods return DataType object. Why do we need to cast them?

Reply:
Good point. Minna and I missed that.